index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>International Mexican Food Convention</title>
+</head>
+
+<body>
+    <header>
```

Comment:
- [ ] You did a great job fixing the issues. :+1: Let's work on some more to match the design. Kindly align the CC campaign button in the lower navigation bar with the Logout button above as suggested by the previous reviewer and presented in the design template. 🤝️

**Your design**
![Screenshot from 2023-07-07 07-30-52](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/17243325/61a44709-e653-4047-8a99-d73f8b375579)


**Template**
![Screenshot from 2023-07-07 07-32-34](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/17243325/38c4fc83-634e-4d45-bc26-065a83decaf0)